lib/liftoff/project.rb

```diff
@@ -63,6 +64,18 @@ def set_company_name(company)
       xcode_project.root_object.attributes['ORGANIZATIONNAME'] = company
     end
 
+    def create_build_configurations(build_configurations)
+      builder = BuildConfigurationBuilder.new(xcode_project)
+      build_configurations ||= []
+
+      build_configurations.each do |configuration|
```

Comment:
What do you think about providing a method on the `BuildConfigurationBuilder` that accepts an array of build configurations and adds them to the project? Pretty much copy and pasting this section into there